src/common.ts

```diff
@@ -4,8 +4,11 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+/* eslint-disable no-console */
```

Comment:
Why disable `no-console`?

Reply:
that was a leftover to figure out that whole "pass local, fail CI" problem yesterday.  Good catch!